File: 2st/Riyeon/평행.py
def solution(dots):
    answer = 0
    if func(dots[0], dots[1]) == func(dots[2], dots[3]):
        answer = 1
    if func(dots[0], dots[2]) == func(dots[1], dots[3]):
        answer = 1
    if func(dots[0], dots[3]) == func(dots[1], dots[2]):
        answer = 1
    return answer

def func(d1, d2):
    return (d2[1]-d1[1]) / (d2[0]-d1[0])

File: 4st/Eunseong/평행.java
class Solution {
    public int solution(int[][] dots) {
        if ((Math.abs((double)dots[0][0] - dots[1][0]) / Math.abs(dots[0][1] - dots[1][1]) == Math.abs((double)dots[2][0] - dots[3][0]) / Math.abs(dots[2][1] - dots[3][1])))
        {
            return 1;
        } else {
            return 0;
        }
    }
}